### DISCUSSION TITLE
Prise en charge de la compression

### DISCUSSION THREAD

**Discussion by julienrf** [cnumr/EcoIndex#207] [Bug]
L’EcoIndex indique que le volume de données transférées est de 2.6 Mo, mais quand je fais une requête depuis mon navigateur je vois 1.9 Mo transféré (grâce à la compression. Le volume décompressé est bien 2.6 Mo). Est-ce que EcoIndex accepte les données compressées ? En tient-il compte ?

**Comment by vvatelot** [comment]
Salut @julienrf , on mesure bien les données compressées transéférées. Tu trouveras la méthode précise ici https://www.ecoindex.fr/comment-ca-marche/#m%C3%A9thodologie-danalyse

Le delta peut s'expliquer de plusieurs façon :
- Utilises tu Chrome ? (version 107 dans le cas d'ecoindex). On a noté des différences entre FF et Chrome sur cette mesure
- Lorsque tu charges ta page, as tu vidé ton cache ?
- Suis tu le même protocole qu'ecoindex ? (chargement de la page, attente 3 sec, scroll bas de page et à nouveau attente 3 sec) ?
- As tu pris en compte la taille du HTML également ?

Si jamais tu veux me donner l'url de la page en question, je peux jeter un oeil...

**Reply by julienrf** [reply]
Bonjour, merci pour ta réponse. En fait c’est une page web qui affiche une carte dans toute la fenêtre, donc j’imagique que quand EcoIndex « scrolle tout en bas » ça dézoome et ça charge de nouveaux tiles.

**Reply by vvatelot** [reply]
Si tu veux, on peut tester ce qu'il se passe... On peut échanger sur le chat https://chat.greenit.eco/ ou tu peux tester tout seul si tu gères un peu python ?

**Reply by julienrf** [reply]
Coucou, merci pour ton message. Je ne pense pas que ce soit nécessaire pour cette discussion. À mon avis les valeurs mesurées sont justes, maintenant que je comprends le protocole.
